doc/source/learning/airship.rst

```diff
@@ -0,0 +1,100 @@
+Airship
+=====================
+
+Airship is a platform that takes YAML documents and turns them into Clouds.
+Takes helm charts, baremetal, security policies and network description and creates a cloud
+
+.. image:: https://airship-treasuremap.readthedocs.io/en/latest/_images/architecture.png
+
+* Airship architecture
+    https://airship-treasuremap.readthedocs.io/en/latest/
+
+* github
```

Comment:
The official links are on opendev. github is just a mirror. Gerrit is used for reviews.

Reply:
Fixing